season_pass/api/__init__.py

```diff
@@ -101,6 +101,19 @@ def invalid_claim(sess: Session = Depends(session)):
     return JSONResponse(status_code=200, content="No invalid claims found.")
 
 
+@router.get("/failure-claim")
+def failure_claim(sess: Session = Depends(session)):
+    now = datetime.now(tz=timezone.utc)
+    failure_claim_list = sess.scalars(select(Claim).where(
+        Claim.created_at <= now - timedelta(minutes=5),
```

Comment:
Failed Tx is failed with no time gap. We don't need to check time interval to check failed Tx.